WooCommerce/Classes/ViewRelated/Orders/Order Creation/Synchronizer/LocalOrderSynchronizer.swift

```diff
@@ -112,3 +112,14 @@ private extension LocalOrderSynchronizer {
         return input.updating(id: Int64(UUID().uuidString.hashValue))
     }
 }
+
+// MARK: Order Helpers
+private extension Order {
+    /// Removes the `itemID` values from items.
+    /// This is needed to create the item without the random generated ID, the remote API would fail otherwise.
+    func removingItemIDs() -> Order {
+        copy (
+            items: items.map { $0.copy(itemID: .zero) }
```

Comment:
Rather than just removing the item ID, I think we need to sanitize the items the same way we do in the remote synchronizer, e.g. by returning this:

```suggestion
            items: items.map { $0.copy(itemID: .zero, subtotal: "", total: "") }
```

Otherwise, order creation works but the wrong item price may be used (not taking into account when `prices_include_tax` is set).

Reply:
Makes sense, does that means that the current release does not handle `prices_include_tax` right now?

Reply:
The current release should handle it, because up until the change that caused the current bug we were only sending the product ID and quantity to the API. Now that we're sending the whole order item we need to be sure to exclude the subtotal/total to handle `prices_include_tax` (along with the ID fix).